ui/tslint.json

```diff
@@ -1,8 +1,16 @@
 {
```

Comment:
Interesting stuff. Was it added automatically? I guess the `tslint.json` doesn't affect us anyway since we're using eslint

Reply:
Yes, it was added automatically. This happens during the 'migrations of packages' process.